phpBB/install/index.html

```diff
@@ -1,11 +1,11 @@
 <html>
 <head>
 <title></title>
-<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+<meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
```

Comment:
The original line is correct HTML 4.0.1, if you want to change it to UTF-8 pls do

<meta http-equiv="content-type" content="text/html; charset=UTF-8">

Instead if you want to port it to HTML5 and UTF-8 pls do

`<meta charset="UTF-8">`

Pls note there are at leat 22 other occurencies in phpBB.